src/dvclive/live.py

```diff
@@ -403,6 +404,16 @@ def end(self):
             finally:
                 mark_dvclive_only_ended()
 
+        if self._dvc_repo and not self._inside_dvc_exp:
+            from dvc.exceptions import DvcException
+
+            try:
+                path = os.path.join(self._dvc_repo.root_dir, "dvc.yaml")
+                yaml = get_dvc_stage_template(self)
+                logger.info(f"To run with DVC, add this to {path}:\n{yaml}")
```

Comment:
Maybe we should also mention the need for `dvc remove {out}`, as we are listing the `live._outs`

Reply:
Although DVC would already hint that whith the new error message. NVM

Reply:
🤔 It's an interesting idea. I worry about adding more comments like this though, so maybe let's wait and see how many people complain? I tried to improve the errors in https://github.com/iterative/dvc/pull/9138 so that it's at least more obvious how to fix the error when you encounter it.